projects/aws/eks-anywhere-packages/Makefile

```diff
@@ -1,6 +1,12 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+PROJECT_COMMIT_HASH:=$(shell git -C eks-anywhere-packages/ rev-parse --short HEAD)
+TIME:=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
We tend to not add times to our builds since it will break reproducibility and the checksum validation. Usually we would use the time from the commit.

Reply:
Yea I can use the ts of the commit. That makes sense.